Automattic/jetpack
# VideoPress: Add VideoStorageMeter component

Closes #25751

#### Changes proposed in this Pull Request:

* Add `VideoStorageMeter` and `ProgressBar` components to VideoPress Package
* Add js tests to VideoPress Package
* Fix https://github.com/Automattic/jetpack/pull/25876#pullrequestreview-1087535452

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Confirm that the default story of `Packages/VideoPress/Video Storage Meter` component works as expected
* Confirm that the default story of `Packages/VideoPress/Progress Bar` component works as expected
* Run `jetpack test packages/videopress js` and confirm test passes

![Screenshot_2022-08-30_14-41-25](https://user-images.githubusercontent.com/8486249/187506460-9cfb1473-8248-488a-8de3-ba78a6f11b90.png)
